package_drupal_script.sh

```diff
@@ -6,7 +6,7 @@ set -o nounset
 
 # Maximise compression
 export XZ_OPT=-9e
-export GZIP=-9e
+export GZIP=-9
```

Comment:
This change worked for me on linux.

Reply:
I don't understand why gzip settings are being changed in this PR. Isn't this about a port change? It's best to stick to one thing.